Mergin/validation.py

```diff
@@ -253,9 +254,14 @@ def check_value_relation(self):
                 ws = layer.editorWidgetSetup(i)
                 if ws and ws.type() == "ValueRelation":
                     cfg = ws.config()
-                    child_layer = next((l for l in self.layers.values() if l.source() == cfg["LayerSource"]), None)
+                    if "Layer" not in cfg or "Key" not in cfg:
```

Comment:
Where "Key" is used in the for loop below?

Reply:
Right.